src/main/java/org/quiltmc/enigma_plugin/index/JarIndexer.java

```diff
@@ -52,11 +54,12 @@ public JarIndexer withContext(EnigmaServiceContext<JarIndexerService> context) {
 		this.disableConstructorParametersIndexing = Arguments.isDisabled(context, Arguments.DISABLE_CONSTRUCTOR_PARAMS);
 		this.disableGetterSetterIndexing = Arguments.isDisabled(context, Arguments.DISABLE_GETTER_SETTER);
 
-		List<String> codecs = context.getArgument(Arguments.CUSTOM_CODECS).map(s -> List.of(s.split(",[\n ]*")))
+		List<String> codecs = context.getArgument(Arguments.CUSTOM_CODECS)
+				.map(e -> e.map(s -> List.of(s.split(",[\n ]*")), Function.identity()))
```

Comment:
why does this still use a split function instead of breaking the current QM profile and using the new `Optional<List<String>> foo = indexers.get(0).getArgument("foo").flatMap(e -> e.right());` method?

Reply:
I thought it'd be better to keep backwards compatibilty, though it isn't that useful to keep

Reply:
I really don't see any point in avoiding breakage

Reply:
Just removed it